### DISCUSSION TITLE
Please add this hotkey option

### DISCUSSION THREAD

**Discussion by mikuppy** [Alex313031/thorium#585] [Ideas]
![image](https://github.com/Alex313031/thorium/assets/3075742/c6bf69c3-514e-40bd-9cee-06debbc5f2e3)
I would be great if you can add this option. This one also
![image](https://github.com/Alex313031/thorium/assets/3075742/cba0d585-2430-4665-96e1-08bbb17dd371)
P/s: this is from Cent browser which is old but i think if thorium can add this is really good.

**Comment by gz83** [comment]
The "boss key" feature has been introduced in the following commit; it might be better to leave other features to extensions.

https://github.com/gz83/thorium/commit/bda54ef4d71f2927cbf90a7c4b0887778c403c74

**Comment by gz83** [comment]
Since the cross-platform implementation of the boss key function is quite complex and has certain maintenance costs, I have decided not to introduce this function.
